site/docs/community.md

```diff
@@ -0,0 +1,43 @@
+# Community
+
+Nessie is developed as a consensus-driven open source product under the Apache 2.0
+license. Development is done in the open leveraging GitHub issues, PRs and using
+Google Groups as a mailing list.
+
+## Get In Touch
+
+[Slack Channel](mailto:slack-subscribe@projectnessie.org)
+: The developers on Nessie frequent the nessie-public Slack channel. You can get an
+  invite to the channel by emailing [slack-subscribe@projectnessie.org](mailto:slack-subscribe@projectnessie.org).
+  If you want your organization invited to the channel, please state that in the request.
+  Whether you're super excited about development or just want to hear what is happening,
+  everyone is welcome to join.
+
+[Google Group](https://groups.google.com/g/projectnessie)
+: If long form is more your thing, we also have created a mailing list on Google groups
+  that you can subscribe to.
+
+[GitHub Issues](https://github.com/projectnessie/nessie/issues)
+: Nessie is developed via GitHub issues and pull requests. If you see a problem
+or want to enhance the product, we suggest you file a GitHub issue for developers to
+review.
+
+[Twitter](https://twitter.com/projectnessie)
+: The [@projectnessie](https://twitter.com/projectnessie) account on Twitter is our official account. Follow-up to keep
+  to date on what is happening with Project Nessie!
+
+[YouTube Channel](https://www.youtube.com/channel/UC5xjzYuGGuGPCY9FNtqZMsQ)
+: Video content for Nessie will be hosted on our YouTube channel.
+
+[Docs](https://github.com/projectnessie/nessie/tree/main/site/docs)
+: Our website is all maintained in our source repository. If there is something you think
+  can be improved, feel free to fork our repository and post a pull request.
+
+[Reviewable](https://reviewable.io/)
+: We use Reviewable (in addition to GitHub reviews) to collaborate on code and docs. They are
+  a great service and support OSS projects.
+
+## Contribution
```

Comment:
I think it would be good to link to https://github.com/projectnessie/nessie/blob/main/CONTRIBUTING.md here as well

Reply:
fixed